.github/dependabot.yml

```diff
@@ -11,119 +11,68 @@ updates:
       interval: "weekly"
       time: "06:00"
     open-pull-requests-limit: 5 
+    groups:
```

Comment:
Why is it necessary to add this directive?

Reply:
Thanks Arve, will remove it for now, will rediscuss with the team if necessary